data/scenarios/Challenges/_dimsum/patron.sw

```diff
@@ -0,0 +1,24 @@
+def modifyCart = \cartType. \device.
```

Comment:
It irks me that we have to duplicate code like this.  I am very motivated to work on #495 .